tools/config/seed.config.ts

```diff
@@ -70,7 +70,7 @@ export class SeedConfig {
    * The default path is `/`, which can be overriden by the `--base` flag when running `npm start`.
    * @type {string}
    */
-  APP_BASE = argv['base'] || '/';
+  APP_BASE = argv['base'] || (this.ENV === ENVIRONMENTS.DEVELOPMENT ? '/' : '');
```

Comment:
Is it confirmable, that this change will not only "fix" the subsite issue for deploying on IIS, but also NOT breaking current deployments on non-IIS servers, or when no subsite structure is used?